lib/Distribution/Sle/15_current.pm

```diff
@@ -21,6 +21,7 @@ use Installation::License::Sle::Firstboot::LicenseAgreementController;
 use Installation::ProductSelection::ProductSelectionController;
 use Installation::Registration::RegistrationController;
 use Installation::ModuleSelection::ModuleSelectionController;
+use Installation::ModuleSelection::v2::ModuleSelectionController;
```

Comment:
I would not use something abstract in the name of Controller/Page (i.e. `v2`), because it does not reflect the reason of difference between those pages. Probably it is better to call it similar to what Joaquin suggested for test modules, e.g. `ModuleRegistrationController`, as on Full medium it allows to register it there, as far as I understand. @jknphy WDYT?